### DISCUSSION TITLE
How can I change the subtitle to only show a selected amount of words?

### DISCUSSION THREAD

**Discussion by SimoneFelici** [cotes2020/jekyll-theme-chirpy#1516] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I want the subtitle to only show like half of the words, just the first line in the article if it's possible.
Also if you can just manually adding it like the tiltle. ex. "subtitle: hi!"
![Web capture_28-1-2024_165812_simonefelici github io](https://github.com/cotes2020/jekyll-theme-chirpy/assets/57272391/42e8f266-416e-4dd9-95f0-373cc58b66d9)
Or just the first line, is it possible

### Operations you have already tried

I tried to put subtitle: and the phrase that I wanted to show,
and couldn't find anything on the wiki

**Comment by kungfux** [comment, marked as answer]
It may not be so easy to display only a certain number of words while making sure they all fit on a single line, because this theme supports responsive design.

But something can be done. If I understand you correctly, you want to display only a single line. This can be done by overriding the styles in `assets\css\jekyll-theme-chirpy.scss' like this

- Option 1
```scss
%text-clip {
  -webkit-line-clamp: 1;
}
```

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/10548881/3627c59c-9d6a-4509-9cdc-97ce1d406730)

- Option 2
```scss
%text-clip {
  -webkit-line-clamp: 1;
  white-space: nowrap;
}
```

![image](https://github.com/cotes2020/jekyll-theme-chirpy/assets/10548881/5064746b-f2c9-4036-b016-ff70fb238188)

**Reply by SimoneFelici** [reply]
Thank you so much, it worked!!!
